README.md

```diff
@@ -2,6 +2,10 @@
 
 Typescript ESLint shareable config
 
+This config intends to provide a list of recommended rules that can be reused across projects.
```

Comment:
This config provides an explicit list of linting rules so you can use the same rules across different projects.

Reply:
Done.